datumaro/plugins/transforms.py

```diff
@@ -50,9 +50,9 @@ def transform_item(self, item):
         if not segments:
             return item
 
-        if not item.has_image:
+        if not item.media:
```

Comment:
As for converters, I think, transforms should check input media type and fail, if it is not supported.